pkg/nodeconfig/payload/payload.go

```diff
@@ -140,7 +140,10 @@ $cni_template=$cni_template.Replace("provider_address",$provider_address)
 # Compare CNI config with existing file, and replace if necessary
 $existing_config=""
 if(Test-Path -Path CNI_CONFIG_PATH) {
-` + "    $existing_config=((Get-Content -Path \"CNI_CONFIG_PATH\" -Raw) -Replace \"`r\",\"\")" + `
+    $config_file_content=(Get-Content -Path CNI_CONFIG_PATH -Raw)
```

Comment:
Shouldn't an existing file without content throw an error?

Reply:
That is not the case
```
PS C:\Users\Administrator> rm /k/cni/config/cni.conf; New-Item /k/cni/config/cni.conf -type file


    Directory: C:\k\cni\config


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        1/30/2024   5:23 PM              0 cni.conf


PS C:\Users\Administrator> cat /k/cni/config/cni.conf
PS C:\Users\Administrator> $foo=(cat /k/cni/config/cni.conf)
PS C:\Users\Administrator> $foo
PS C:\Users\Administrator> $foo.gettype()
You cannot call a method on a null-valued expression. 
At line:1 char:1
+ $foo.gettype()
+ ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

```